frontend/pages/hosts/details/HostDetailsPage/HostActionsDropdown/helpers.tsx

```diff
@@ -176,7 +175,7 @@ const canWipeHost = ({
   );
 };
 
-const canUnlock = ({
+export const canUnlock = ({
```

Comment:
are we ok exporting things for tests?

Reply:
We are testing the host actions at the integration layer. We don't export the helper methods. We render the host actions dropdown component itself and set up the testing environment to test a scenario. If you look in HostActionsDropdown.tests.tsx you can see examples. We do this as it mimics more how a real user would use the UI.